### ISSUE TITLE
Is It possible to prevent popup from closing on select?

### ISSUE THREAD

**Issue by AlexKhymenko** [ng-bootstrap/ng-bootstrap#2397]
### Bug description:
I know there is an issue regarding a lot of items and that library wants to go way of google and amazon to show only 10 items.
In this case i just want to give user at least option to see that there are more options and when he presses it it will load full list or add 10 more items to list. Im adding specific option load 10 more to the list.
The problem is i cant get typeahead from not closing when user selects specific option. 
Is It possible to prevent popup from closing on select?

Angular: 6.02

ng-bootstrap: 2.0.0

Bootstrap:

**Comment by pkozlowski-opensource** [comment]
Nope, we don't have such functionality. Marking as a feature request.
